docs/api_reference/_autosummary/flax.linen.BatchNorm.rst

```diff
@@ -0,0 +1,20 @@
+﻿flax.linen.BatchNorm
```

Comment:
Regarding all the `/api_reference/*.rst` files: are these autogenerated? if yes, consider mentioning them in the PR (I don't immediately see why we have different paths, and why so many new files appeared)

Reply:
Yes, thanks for the heads up, they have been removed!